src/utils/yelpFusionApiUtils.ts

```diff
@@ -19,18 +19,18 @@ export class YelpFusionApiUtils {
 				latitude: geocode.lat,
 				longitude: geocode.lng,
 			})
-			.then((response) => response.jsonBody.businesses[0].id)
-			.then((id) => this.yelpClient.business(id))
-			.then((businessResult) =>
+			.then((response: any) => response.jsonBody.businesses[0].id)
```

Comment:
You really should know the shape of the expected response... either generated from the json (use a json to ts tool) or if you are using Graphql Mesh it does all that for you.  Rarely should you have to use `any`.